proposals/0005-dns.md

```diff
@@ -59,12 +59,14 @@ Users have multiple options for creating a domain-name mapping.
 ## DNS TXT record
 [usage-dns-txt-record]: #usage-dns-txt-record
 
-The first option is to set a DNS TXT record at the domain which maps to a "key"-addressed Dat URL. The client will lookup this TXT record first and load the resulting Dat. That record should fit the following schema:
+The first option is to set a DNS TXT record at the domain which maps to a "key"-addressed Dat URL. The client will lookup this TXT record first and load the resulting Dat. That record should contain the following schema:
 
 ```
 datkey={key}
```

Comment:
Could we clarify that the key in this case must be 64 hex characters?

Reply:
Ok I have added that to the note. on line 68.